docs/core/tools/rid-specific-tools.md

```diff
@@ -0,0 +1,214 @@
+
+- **RID-specific**: Compiled for particular operating systems and architectures.
+- **Self-contained**: Include the .NET runtime and don't require a separate .NET installation.
+- **Native AOT**: Use Ahead-of-Time compilation for faster startup and smaller memory footprint.
+
+When users install a RID-specific tool, the .NET CLI automatically selects and installs the appropriate package for their platform.
+
+## Opt in to RID-specific packaging
+
+To create a RID-specific tool, configure your project with one of the following MSBuild properties:
+
+### RuntimeIdentifiers property
+
+Use `RuntimeIdentifiers` to specify the platforms your tool supports:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net10.0</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>mytool</ToolCommandName>
+    <RuntimeIdentifiers>win-x64;linux-x64;osx-arm64</RuntimeIdentifiers>
+  </PropertyGroup>
+</Project>
+```
+
+### ToolPackageRuntimeIdentifiers property
+
+Alternatively, use `ToolPackageRuntimeIdentifiers` for tool-specific RID configuration:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net10.0</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>mytool</ToolCommandName>
+    <ToolPackageRuntimeIdentifiers>win-x64;linux-x64;osx-arm64</ToolPackageRuntimeIdentifiers>
+  </PropertyGroup>
+</Project>
+```
+
+Use a semicolon-delimited list of RID values. For a list of Runtime Identifiers, see the [RID catalog](../rid-catalog.md).
+
+## Package your tool
+
+The packaging process differs depending on whether you're using AOT compilation. To build a NuGet package, or *.nupkg* file from the project, run the [dotnet pack](dotnet-pack.md) command.
+
+### RID-specific and self-contained tools
+
+For tools without AOT compilation, run `dotnet pack` once:
+
+```dotnetcli
+dotnet pack
+```
+
+This command creates multiple NuGet packages:
+
+- One package for each RID: `<packageName>.<RID>.<packageVersion>.nupkg`
+  - Example: `mytool.win-x64.1.0.0.nupkg`
+  - Example: `mytool.linux-x64.1.0.0.nupkg`
+  - Example: `mytool.osx-arm64.1.0.0.nupkg`
+- One RID-agnostic pointer package: `<packageName>.<packageVersion>.nupkg`
+  - Example: `mytool.1.0.0.nupkg`
+
+### AOT tools
+
+For tools with AOT compilation (`<PublishAot>true</PublishAot>`), you must pack separately for each platform:
+
+- Pack the top-level package once (on any platform):
+
+   ```dotnetcli
+   dotnet pack
+   ```
+
+- Pack for each specific RID on the corresponding platform:
+
+   ```dotnetcli
+   dotnet pack -r win-x64
+   dotnet pack -r linux-x64
+   dotnet pack -r osx-arm64
+   ```
+
+   You must run each RID-specific pack command on the matching platform because AOT compilation produces native binaries.
```

Comment:
What do you think about linking to https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot here for further guidance about NAOT publishing? I was thinking of a call to action like

> Learn more about the prerequisites for Native AOT compilation at  [Native AOT deployment](https://learn.microsoft.com/dotnet/core/deploying/native-aot).

What do you think?